application/comit_node/src/http_api/rfc003/action.rs

```diff
@@ -302,15 +323,17 @@ impl IntoResponseBody for ethereum::SendTransaction {
         let ethereum::SendTransaction {
             to,
             data,
-            value,
+            amount,
             gas_limit,
+            network,
         } = self;
         match query_params {
-            GetActionQueryParams::None {} => Ok(ActionResponseBody::SendEthereumTransaction {
-                to: Some(to),
+            GetActionQueryParams::None {} => Ok(ActionResponseBody::EthereumInvokeContract {
+                contract_address: to,
                 data,
-                value,
+                amount,
```

Comment:
🔧  Rename this field value.

In both Bitcoin and Ethereum communities it's called`value`.

Reply:
The spike said amount...

Reply:
I chose `amount` because it seems less ambiguous and more consistent across multiple blockchains.